### ISSUE TITLE
Cisco 2360 says invalid port for 10g interfaces 

### ISSUE THREAD

**Issue by garbled1** [slub/switchconfig#1]
example output of sh int

```
TenGigabitEthernet0/4 is down, line protocol is down (notconnect) 
  Hardware is Ten Gigabit Ethernet, address is 6c50.4d17.f1b8 (bia 6c50.4d17.f1b8)
  Description: None
  MTU 1500 bytes, BW 10000000 Kbit/sec, DLY 10 usec, 
     reliability 255/255, txload 1/255, rxload 1/255
  Encapsulation ARPA, loopback not set
  Keepalive not set
  Full-duplex, 10Gb/s, link type is auto, media type is Not Present
  input flow-control is off, output flow-control is unsupported 
  ARP type: ARPA, ARP Timeout 04:00:00
  Last input never, output never, output hang never
  Last clearing of "show interface" counters never
  Input queue: 0/75/0/0 (size/max/drops/flushes); Total output drops: 0
  Queueing strategy: fifo
  Output queue: 0/40 (size/max)
  5 minute input rate 0 bits/sec, 0 packets/sec
  5 minute output rate 0 bits/sec, 0 packets/sec
     0 packets input, 0 bytes, 0 no buffer
     Received 0 broadcasts (0 multicasts)
     0 runts, 0 giants, 0 throttles
     0 input errors, 0 CRC, 0 frame, 0 overrun, 0 ignored
     0 watchdog, 0 multicast, 0 pause input
     0 input packets with dribble condition detected
     0 packets output, 0 bytes, 0 underruns
     0 output errors, 0 collisions, 2 interface resets
     0 unknown protocol drops
     0 babbles, 0 late collision, 0 deferred
     0 lost carrier, 0 no carrier, 0 pause output
     0 output buffer failures, 0 output buffers swapped out

```

**Comment by garbled1** [comment]
```
xxx>sh int status

Port      Name               Status       Vlan       Duplex  Speed Type
Gi0/1                        notconnect   2            auto   auto 10/100/1000BaseTX
Gi0/2                        notconnect   2            auto   auto 10/100/1000BaseTX
Gi0/3                        notconnect   2            auto   auto 10/100/1000BaseTX
Gi0/4                        notconnect   2            auto   auto 10/100/1000BaseTX
Gi0/5     None               notconnect   2            auto   auto 10/100/1000BaseTX
Gi0/6     None               notconnect   2            auto   auto 10/100/1000BaseTX
Gi0/7     None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/8     None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/9     None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/10    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/11    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/12    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/13    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/14    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/15    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/16    None               connected    1          a-full a-1000 10/100/1000BaseTX
Gi0/17    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/18    None               connected    1          a-full a-1000 10/100/1000BaseTX
Gi0/19    None               connected    1          a-full a-1000 10/100/1000BaseTX
Gi0/20    None               connected    1          a-full  a-100 10/100/1000BaseTX
Gi0/21    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/22    None               connected    1          a-full a-1000 10/100/1000BaseTX
Gi0/23    None               notconnect   1            auto   auto 10/100/1000BaseTX
          
Port      Name               Status       Vlan       Duplex  Speed Type
Gi0/24    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/25    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/26    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/27    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/28    None               connected    1          a-full  a-100 10/100/1000BaseTX
Gi0/29    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/30    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/31    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/32    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/33    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/34    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/35    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/36    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/37    None               connected    1          a-full a-1000 10/100/1000BaseTX
Gi0/38    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/39    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/40    None               connected    1          a-full  a-100 10/100/1000BaseTX
Gi0/41    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/42    None               connected    1          a-half   a-10 10/100/1000BaseTX
Gi0/43    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/44    None               connected    1          a-full a-1000 10/100/1000BaseTX
Gi0/45    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/46    None               connected    1          a-full  a-100 10/100/1000BaseTX
Gi0/47    None               notconnect   1            auto   auto 10/100/1000BaseTX
Gi0/48    None               notconnect   1            auto   auto 10/100/1000BaseTX
          
Port      Name               Status       Vlan       Duplex  Speed Type
Te0/1     None               notconnect   1            full    10G Not Present
Te0/2     None               notconnect   1            full    10G Not Present
Te0/3     None               connected    1            full    10G SFP-10GBase-CX1
Te0/4     None               notconnect   1            full    10G Not Present
Fa0       None               notconnect   routed       auto   auto 10/100BaseTX


```

**Comment by schorschii** [comment]
The fix mentioned in #1 fixes this too. You can now add "Te" interfaces to the config:
```
const VISIBLE_PORTS = ['Gi', 'Fa', 'Te'];
```
Please have a try with the current master branch.

**Comment by garbled1** [comment]
Works great, thanks!
